Sum-of-numbers/getSum.js

```diff
@@ -0,0 +1,13 @@
+const getSum = (a, b) => {
```

Comment:
you dont need to use `else if` there because you don't need to enter second condition
maybe try to use ternary operator instead